### PR TITLE
Fixed job life cycle event listener exception on log

### PR DESCRIPTION
**Pull request by hqin2016**
Tested in holdem 6 Kubernetes pods, the exception is not appearing after the fix.

### CODE REVIEW

File: azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java [hunk 1]
```diff
@@ -699,12 +702,16 @@ private void uploadLogFile(final int execId) {
    */
   @Override
   public void handleEvent(final Event event) {
-    final FlowRunner flowRunner = (FlowRunner) event.getRunner();
-    final ExecutableFlow flow = flowRunner.getExecutableFlow();
-    // Set Flow level SLA options for containerized executions
-    this.triggerManager
-        .addTrigger(flow.getExecutionId(), SlaOption.getFlowLevelSLAOptions(flow
-            .getExecutionOptions().getSlaOptions()));
+    final List<EventType> flowEventTypeList = Arrays.asList(EventType.FLOW_STARTED,
```
Comment (jakhani) [inline azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java:705]: Why don't you take this in enum? Introduce method like isFlowEvent(eventType)

**Review by djaiswal83 — APPROVED** [review]
Thanks for addressing this quickly.

File: azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java (hunk 1 above)
Comment (hqin2016) [reply azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java]: good idea

**Comment by hqin2016** [comment]
tested on holdem 4 for sanity check, and holdem 6 Kubernetes pod

**Review by djaiswal83 — APPROVED** [review]
This is much cleaner code. Excellent suggestion @jakhani

**Review by jakhani — APPROVED** [review]

**Pull request merged**